src/components/menuBar/menuBarButtons.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { SetStateAction, Dispatch } from "react";
+import styled from "@emotion/styled";
+import { Tag } from "../displayDocuments/documents";
+
+const Button = (props: { title: string; selected: boolean }): JSX.Element => {
```

Comment:
Usually we will destructure the props here

```suggestion
const Button = ({title, selected}: { title: string; selected: boolean }): JSX.Element => {
```